lib/util.js

```diff
@@ -779,14 +764,15 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
 }
 
 function formatPromise(ctx, value, recurseTimes, visibleKeys, keys) {
-  var output = [];
-  var internals = inspectPromise(value);
-  if (internals.status === 'pending') {
+  const output = [];
+  const [state, result] = binding.getPromiseDetails(value);
+
+  if (state === binding.kPending) {
     output.push('<pending>');
   } else {
     var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
-    var str = formatValue(ctx, internals.value, nextRecurseTimes);
-    if (internals.status === 'rejected') {
+    var str = formatValue(ctx, result, nextRecurseTimes);
+    if (state === binding.kRejected) {
```

Comment:
@Fishrock123 this set of APIs is relatively new, having been added in V8 5.7 in https://github.com/v8/v8/commit/28432588c231019c2bb0dfbba8eba4304779706b. That might be why.